libraries/linux_updates.rb

```diff
@@ -158,65 +159,66 @@ def extract_xml_updates(updates_el)
-echo -n '{"available":['
-DEBIAN_FRONTEND=noninteractive apt upgrade --dry-run | grep Inst | tr -d '[]()' |\\
-  awk '{ printf "{\\"name\\":\\""$2"\\",\\"version\\":\\""$4"\\",\\"repo\\":\\""$5"\\",\\"arch\\":\\""$6"\\"}," }' | rev | cut -c 2- | rev | tr -d '\\n'
-echo -n ']}'
+    debian_updates = debian_base + <<~PRINT_JSON
+      echo -n '{"available":['
+      DEBIAN_FRONTEND=noninteractive apt upgrade --dry-run | grep Inst | tr -d '[]()' |\\
+        awk '{ printf "{\\"name\\":\\""$2"\\",\\"version\\":\\""$4"\\",\\"repo\\":\\""$5"\\",\\"arch\\":\\""$6"\\"}," }' | rev | cut -c 2- | rev | tr -d '\\n'
+      echo -n ']}'
     PRINT_JSON
     parse_json(debian_updates)
   end
 
   private
 
   def debian_base
-    base = <<-PRINT_JSON
-#!/bin/sh
-COMMAND="DEBIAN_FRONTEND=noninteractive apt update >>/dev/null 2>&1"
-eval $COMMAND
-while [ $? -ne 0 ]
-do
-sleep 30s
-eval $COMMAND
-done
-echo " "
+    <<~PRINT_JSON
+      #!/bin/sh
+      COMMAND="DEBIAN_FRONTEND=noninteractive apt update >>/dev/null 2>&1"
+      eval $COMMAND
+      while [ $? -ne 0 ]
+      do
+      sleep 30s
+      eval $COMMAND
+      done
+      echo " "
     PRINT_JSON
-    base
   end
 end
 
 class RHELUpdateFetcher < UpdateFetcher
   def packages
-    rhel_packages = <<-PRINT_JSON
-sleep 2 && echo " "
-echo -n '{"installed":['
-rpm -qa --queryformat '"name":"%{NAME}","version":"%{VERSION}-%{RELEASE}","arch":"%{ARCH}"\\n' |\\
-  awk '{ printf "{"$1"}," }' | rev | cut -c 2- | rev | tr -d '\\n'
-echo -n ']}'
+    # rubocop:disable Style/FormatStringToken
+    rhel_packages = <<~PRINT_JSON
+      sleep 2 && echo " "
+      echo -n '{"installed":['
+      rpm -qa --queryformat '"name":"%{NAME}","version":"%{VERSION}-%{RELEASE}","arch":"%{ARCH}"\\n' |\\
+        awk '{ printf "{"$1"}," }' | rev | cut -c 2- | rev | tr -d '\\n'
+      echo -n ']}'
     PRINT_JSON
     parse_json(rhel_packages)
+    # rubocop:enable Style/FormatStringToken
   end
 
   def updates
-    rhel_updates = <<-PRINT_JSON
-#!/bin/sh
-python -c 'import sys; sys.path.insert(0, "/usr/share/yum-cli"); import cli; ybc = cli.YumBaseCli(); ybc.setCacheDir("/tmp"); list = ybc.returnPkgLists(["updates"]);res = ["{\\"name\\":\\""+x.name+"\\", \\"version\\":\\""+x.version+"-"+x.release+"\\",\\"arch\\":\\""+x.arch+"\\",\\"repository\\":\\""+x.repo.id+"\\"}" for x in list.updates]; print "{\\"available\\":["+",".join(res)+"]}"'
+    rhel_updates = <<~PRINT_JSON
+      #!/bin/sh
+      python -c 'import sys; sys.path.insert(0, "/usr/share/yum-cli"); import cli; ybc = cli.YumBaseCli(); ybc.setCacheDir("/tmp"); list = ybc.returnPkgLists(["updates"]);res = ["{\\"name\\":\\""+x.name+"\\", \\"version\\":\\""+x.version+"-"+x.release+"\\",\\"arch\\":\\""+x.arch+"\\",\\"repository\\":\\""+x.repo.id+"\\"}" for x in list.updates]; print "{\\"available\\":["+",".join(res)+"]}"'
     PRINT_JSON
     cmd = @inspec.bash(rhel_updates)
-    unless cmd.exit_status == 0
+    unless cmd.exit_status.zero?
```

Comment:
nit: similar as above. I am not sure what the value is? its even more characters